### ISSUE TITLE
Xournal++ crash

### ISSUE THREAD

**Issue by cliverlong-1961** [xournalpp/xournalpp#6154]
### Operating System

Mac OS

### (Linux Only) Display Server

None

### Installation Method

MacOS installation resources from website

### Xournal++ Version

1.2.4

### libgtk Version

3.24.38

### Bug Description

Xournal+ crashed when using "move up" on a page

### Expected Behaviour

No crash

### Steps to Reproduce

1. Click on a page.
2. Select "move up"

### Additional Context

[errorlog.20241211-183550.log](https://github.com/user-attachments/files/18100654/errorlog.20241211-183550.log)

**Comment by rolandlo** [comment]
I'll check if I can reproduce. @cliverlong-1961 Two questions:
- Is this on MacOS ARM or MacOS Intel?
- Which version did you use (the ARM one or the other one)?

**Comment by cliverlong-1961** [comment]
This is on a MacPro 5,1 , Intel Xeon processor.

The Intel version is installed. Surely ARM code would not install or run on intel?

I have run the "move up" command on another Xornal+ file and the problem did not occur.

I will experiment with this and try to reproduce the problem. If it recurs I will add to this bug report.

**Comment by rolandlo** [comment]
> I can't reproduce on Linux. Does it happen everytime or was it a one time thing? @rolandlo Is it something you can observe on your mac?

No, I don't observe it. Maybe I don't have the right xopp-file.
